src/components/single-ticket/ReportTicket.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useState, VoidFunctionComponent } from "react"
+import React, { FC, useState, VoidFunctionComponent } from "react"
```

Comment:
**nitpick:**
- unused import of `FC` (leftover)
- there is `VFC` alias for `VoidFunctionComponent`